### PR TITLE
OCPBUGS#3813: Restarting and shutting down the cluster

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/OCPBUGS-3813

Link to docs preview: https://56519--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/graceful-cluster-shutdown.html#graceful-shutdown_graceful-shutdown-cluster

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/graceful-shutdown.adoc [hunk 1]
```diff
@@ -21,6 +21,14 @@ You can shut down a cluster until a year from the installation date and expect i
 [IMPORTANT]
 ====
 It is important to take an etcd backup before performing this procedure so that your cluster can be restored if you encounter any issues when restarting the cluster.
+
+Even though the cluster is expected to be functional after the restart, the cluster might not recover due to unexpected conditions, for example:
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/graceful-shutdown.adoc]: Addressed

File: modules/graceful-shutdown.adoc [hunk 2]
```diff
@@ -21,6 +21,14 @@ You can shut down a cluster until a year from the installation date and expect i
 [IMPORTANT]
 ====
 It is important to take an etcd backup before performing this procedure so that your cluster can be restored if you encounter any issues when restarting the cluster.
+
+Even though the cluster is expected to be functional after the restart, it might not recover due to unexpected conditions, such as:
```
Comment (ousleyp) [inline modules/graceful-shutdown.adoc:25]: This sentence ("Even though...") is hard to parse and might be misleading. The ISG also states "To avoid possible translation problems, use complete sentences to introduce lists and not phrases." Suggested rephrase:

`For example, the following conditions can cause the restarted cluster to malfunction:`

**Review by ousleyp — COMMENTED** [review]
My main reason for not merging this is the lowercase xref title on line 23 of `backup_and_restore/graceful-cluster-shutdown.adoc`, but there are a couple of other issues that I think would be great to fix in this PR, if possible. Please re-add the `merge-review-needed` label when it's ready to go. Thanks!

File: modules/graceful-shutdown.adoc (hunk 2 above)
Comment (xenolinux) [reply modules/graceful-shutdown.adoc:25]: Addressed

File: backup_and_restore/graceful-cluster-shutdown.adoc [hunk 3]
```diff
@@ -19,3 +19,5 @@ include::modules/graceful-shutdown.adoc[leveloffset=+1]
 .Additional resources
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply backup_and_restore/graceful-cluster-shutdown.adoc]: Addressed
